### PR TITLE
Add clojure-version and *clojure-version*

### PR DESCRIPTION
**Pull request by patrickgombert**

### CODE REVIEW

**Comment by mylesmegyesi** [comment]
?

**Comment by patrickgombert** [comment]
?

**Comment by mylesmegyesi** [comment]
nvm. I didn't know this was in clojure.

**Comment by mylesmegyesi** [comment]
why the `clojure.core` qualifier?

**Comment by patrickgombert** [comment]
To differentiate from current clojure. The doc on qualifier is ambiguous, but either seems to be omitted (on release) or the current branch (on snapshots).

We can use the same strategy if that would be better.

**Comment by mylesmegyesi** [comment]
Well, its just a conventional Maven thing. [This](http://stackoverflow.com/questions/5901378/what-exactly-is-a-maven-snapshot-and-why-do-we-need-it) explains it kind of. I don't think we need it.

**Comment by patrickgombert** [comment]
updated

File: src/clj/clojure/next.clj [hunk 1]
```diff
@@ -9,6 +9,20 @@
             [clojure.lang.random     :refer [rand-float]]
             [clojure.lang.protocols  :refer :all]))
 
+(def ^:dynamic *clojure-version*
+  {:interim     true
+   :major       1
+   :minor       6
+   :incremental 0
+   :qualifier   "clojure.core"})
```
Comment (mylesmegyesi) [inline src/clj/clojure/next.clj]: Do we need this anymore?

**Pull request merged**